src/main.cpp

```diff
@@ -16,6 +16,10 @@ int main(int argc, char *argv[])
     app.setApplicationName("Notes");
     app.setApplicationVersion(APP_VERSION);
 
+#if defined(Q_OS_LINUX) && QT_VERSION >= QT_VERSION_CHECK(5, 7, 0)
```

Comment:
I think this check can be expanded to other *nixes too. And the Qt version check can be removed as well, as we already require at least Qt 5.12.0:

```suggestion
#if defined(Q_OS_UNIX) && !defined(Q_OS_MACOS)
```